test/integration/minishift.go

```diff
@@ -105,6 +110,13 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 }
 
 func (m *Minishift) getOpenShiftUrl() string {
-	cmdOut, _, _ := m.runner.RunCommand("console --url")
+	cmdOut, _, _ := m.runner.RunCommand(ConsoleURL)
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) getRoutingUrl() string {
```

Comment:
But this only works, if there is only one app deployed. This will break once we have multiple routes. One needs to specify the route or application name.